docs/UserGuide.md

```diff
@@ -378,7 +423,7 @@ Examples:
 
 #### 3.9 Clearing all entries : `c-clear`
 
-Clears all entries from the contact list.
+Clears all entries from the employee directory.
```

Comment:
Perhaps use "Employee Directory" instead of "employee directory", since it is referring to a particular section inside tCheck.